src/components/forums/Forum.js

```diff
@@ -190,6 +223,60 @@ const Forum = () => {
         return body;
     }
 
+    // API - Edit your own comments
+        const handleEditComment= (selectedComment) => {
+        if (comment === '') {
+            setCommentError(true);
+            setCommentErrorText('Please enter your comment');
+            return false;
+        } else {
+            //console.log("about to call api edit forum");
+            callApiEditComment(selectedComment);
+            setShowEditAlertMessage(true);
+            setTimeout(() => {
+                window.location.reload();
+            }, 100);
```

Comment:
this refresh delay is too short... we usually use 3 seconds (3000 ms) instead of 0.1 seconds (100 ms)

Reply:
The delete uses 100ms though... so I wanted it to be consistent. But I will change it!

Reply:
I dont think the delete displays an alert message. The other functions that use alert messages do a 3 sec delay